CHANGES.txt

```diff
@@ -1,6 +1,9 @@
 1.3.x (unreleased)
  * Removed use of Thrift describe_ring and replaced it with native Java Driver
    support for fetching TokenRanges (SPARKC-93)
+ * Decoupled PredicatePushDown logic from Spark (SPARKC-166)
```

Comment:
Yes, but you can introduce additional field in Jira for CHANGES purpose. Hence, you do not have have each Jira issue mentioned in CHANGES.